api/internal/plugins/fnplugin/fnplugin.go

```diff
@@ -187,6 +187,7 @@ func (p *FnPlugin) invokePlugin(input []byte) ([]byte, error) {
 	p.runFns.Input = bytes.NewReader(input)
 	p.runFns.Functions = append(p.runFns.Functions, functionConfig)
 	p.runFns.Output = &ouputBuffer
+	p.runFns.WorkDir = p.h.WorkingDir()
```

Comment:
Instead of here, could we set it up front in `NewFnPlugin(o *types.FnPluginLoadingOptions)`? And then not need `(c *PluginHelpers) WorkingDir()`, I think?